vertx-grpc-common/src/main/java/io/vertx/grpc/common/impl/GrpcMessageImpl.java

```diff
@@ -21,10 +21,18 @@ public class GrpcMessageImpl implements GrpcMessage {
 
   private final String encoding;
   private final Buffer payload;
+  private final boolean trailer;
```

Comment:
overall this trailer field should not be present here, instead the encode message should have a parameter to specify the message should be encoded as a trailer for a given message

Reply:
ok